tensorflow/python/keras/callbacks.py

```diff
@@ -1102,11 +1102,12 @@ class ModelCheckpoint(Callback):
         in the filename.
       monitor: quantity to monitor.
       verbose: verbosity mode, 0 or 1.
-      save_best_only: if `save_best_only=True`, the latest best model according
-        to the quantity monitored will not be overwritten.
-        If `filepath` doesn't contain formatting options like `{epoch}` then
-        `filepath` will be overwritten by each new better model.
-      mode: one of {auto, min, max}. If `save_best_only=True`, the decision to
+      save_best_only: if `save_best_only=True`, it only saves when the model 
+        is considered the "best" and the latest best model according to the 
+        quantity monitored will not be overwritten. If `filepath` doesn't contain 
+        formatting options like `{epoch}` then `filepath` will be overwritten 
+        by each new better model.
+      mode: one of `{auto, min, max}`. If `save_best_only=True`, the decision to
```

Comment:
Since auto, min, and max are string, I'd probably say {'auto', 'min', 'max'} here

Reply:
resolved! @rchao